### DISCUSSION TITLE
Config setting to only create PRs that are tied to CVE issues

### DISCUSSION THREAD

**Discussion by sbsamaro** [renovatebot/renovate#15858] [Request Help] [answered]
Is there any config settings to prevent auto generated PRs to only generate if they have a CVE issue.
IE: undo chore prs and fix prs that do not have CVEs tied to it.  If we close these prs without cve issues. if there is a vulnerability with that package it will pick up in the future?

Also is there a bot that will auto approve prs for auto merge?

**Comment by astellingwerf** [comment]
I must admit that it's not a full answer, but you might be interested in #15159

**Comment by rarkins** [comment, marked as answer]
See here: https://github.com/renovatebot/renovate/discussions/15490#discussioncomment-2702421

But perhaps we should turn this into a first-class config option for simplicity sake
